app/services/validator_check_active_service.rb

```diff
@@ -110,4 +102,8 @@ def acceptable_stake?(validator)
     with_acceptable_stake.exists?
   end
 
+  def delinquent_history(validator)
```

Comment:
this method is not the same as the previous one and does not work as it should

Reply:
Fixed.